docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+services:
+
+  # assumes imgsrv-sample-data has been checked out next to "imgsrv"
+  dev:
+    build: .
+    volumes:
+      - .:/htapps/babel/imgsrv
+      - "../imgsrv-sample-data/sdr1:/sdr1"
+      - "../imgsrv-sample-data/etc:/htapps/babel/etc"
+      - "../imgsrv-sample-data/watermarks:/htapps/babel/watermarks"
+      - "../imgsrv-sample-data:/tmp/imgsrv-sample-data"
+    environment:
+      - SDRROOT=/htapps/babel
+      - SDRDATAROOT=/sdr1
+      - HT_DEV=
+      - MARIADB_USER=ht_web
+      - REMOTE_ADDR=127.0.0.1
```

Comment:
Is this accounting for hardcoded assumptions in imgsrv?

Reply:
Setting keys to avoid Perl grousing.